src/rpc/rpc_base_conn.js

```diff
@@ -129,12 +144,37 @@ class RpcBaseConnection extends EventEmitter {
         return this._state === STATE_CLOSED;
     }
 
+    _encode_message(msg) {
+        return msg.encode();
+    }
+
+    _decode_message(encoded_message) {
```

Comment:
jsdoc:
```suggestion
    /**
     * @param {Buffer[]} encoded_message
     * @returns {RpcMessage}
     */
    _decode_message(encoded_message) {
```

Reply:
This will break as this does accept  buffers in all cases, especially for the case of fcall

Reply:
done